allennlp/service/servable/models/decomposable_attention.py

```diff
@@ -0,0 +1,39 @@
+from allennlp.common import Params, constants
+from allennlp.data import Vocabulary
+from allennlp.data.dataset_readers.snli import SnliReader
+from allennlp.data.fields import TextField
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.models import DecomposableAttention
+from allennlp.nn import InitializerApplicator
+from allennlp.service.servable import Servable, JSONDict
+
+class DecomposableAttentionServable(Servable):
+    def __init__(self):
+        constants.GLOVE_PATH = 'tests/fixtures/glove.6B.300d.sample.txt.gz'
+        dataset = SnliReader().read('tests/fixtures/snli_example.jsonl')
+        vocab = Vocabulary.from_dataset(dataset)
+        self.vocab = vocab
+        dataset.index_instances(vocab)
+        self.dataset = dataset
+        self.token_indexers = {'tokens': SingleIdTokenIndexer()}
```

Comment:
Same comments here about explicitly passing the tokenizer and token indexers to the `SnliReader`, just to avoid subtle bugs and/or duplication later.